tests/test_tree_distance.py

```diff
@@ -3,17 +3,86 @@
 import pytest
 
 from cogent3 import make_tree
-from cogent3.phylo.tree_distance import lin_rajan_moret
+from cogent3.phylo.tree_distance import (
+    lin_rajan_moret,
+    matching_cluster_distance,
+    rooted_robinson_foulds,
+    unrooted_robinson_foulds,
+)
+
+ROOTED_DISTANCE_MEASURES = (matching_cluster_distance, rooted_robinson_foulds)
+UNROOTED_DISTANCE_MEASURES = (lin_rajan_moret, unrooted_robinson_foulds)
+
+
+# unrooted rf distance
+def test_unrooted_rf_different_trees():
+    a = make_tree(treestring="(a, b, (c, (d, e)));")
+
+    b = make_tree(treestring="(a, b, (d, (c, e)));")
```

Comment:
add a comment about where the expected values come from, i.e. what makes them "expected"?